README.md

```diff
@@ -1,28 +1,96 @@
-# Introduction
+# Nimble Infrastructure Template
 
-Infrastructure (Terraform) configuration template for projects at Nimble.
+An infrastructure template for web applications powered by Terraform.
 
-## Usage
+**Supported cloud:** AWS. We are looking for contributions to implement support for GCP, Heroku, and Azure!
 
-1. Create a folder name and cd into that folder
+**Supported flavors:**
+- A `Basic` flavor is in the plan, but not available yet
+- The `Complete` flavor generates the following infrastructure:
 
-```bash
-mkdir awesome-infra && cd awesome-infra
-```
+![Diagram of the Complete Infrastructure](https://github.com/nimblehq/infrastructure-templates/blob/feature/55-add-documentation/img/diagram_complete.png?raw=true)
+
+# Usage
 
-2. Start the CLI to generate the infrastructure configuration in Terraform
+## Generate a new project
 
 ```bash
 npm install -g @nimblehq/infra-template
 
 nimble-infra generate {project-name}
 ```
 
-* With npx:
+or
+
 ```bash
 npx @nimblehq/infra-template generate {project-name}
 ```
 
+## Deploy your infrastructure
```

Comment:
As discussed, should we move this part `Deploy your infrastructure` to the `generated project's readme`

Reply:
We can create a Documentation add-on and will copy this content into the needed folder, maybe the wiki or README of the generated project 🤔

Reply:
IMO we do not need a "documentation add-on", it should be placed in the `common` files when a projected variant is created (this documentation is specific to the AWS variant). 

Here is my suggested implementation: 19f1c21 ✅